<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cNavigatorWindow</class>
 <widget class="QDialog" name="cNavigatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1903</width>
    <height>1152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,10,1">
     <item>
      <widget class="QGroupBox" name="groupBoxParameterSet">
       <property name="title">
        <string>Edited parameter set</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollAreaParameterSet">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaParameterSetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>491</width>
             <height>1089</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_navigator_options">
         <property name="title">
          <string>Preview quality options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyComboBox" name="comboBox_navigator_preview_size">
              <item>
               <property name="text">
                <string>Small</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Medium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Big</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Quality</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyComboBox" name="comboBox_navigator_preview_quality">
              <item>
               <property name="text">
                <string>Auto</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1/32</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_navigator_shadows">
              <property name="toolTip">
               <string>Enable shadow calculation in the preview</string>
              </property>
              <property name="text">
               <string>Shadows</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_navigator_reflections">
              <property name="toolTip">
               <string>Enable reflections and transparency calculation in the preview</string>
              </property>
              <property name="text">
               <string>Reflectons</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_navigator_volumetrics">
              <property name="toolTip">
               <string>Enable volumetric effects calculation in the preview</string>
              </property>
              <property name="text">
               <string>Volumetrics</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_navigator_dark_glow">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Disable shadows, reflections and volumetric effects, and enable fast dark glow effect which improves perception of depth.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Dark glow</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="RenderedImage" name="widgetRenderedImage" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_mouse_click_functions">
           <property name="text">
            <string>Mouse click function:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyComboBox" name="comboBox_mouse_click_function">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selection for action which will be executed after clicking on the image&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;No action - blocks functionality of mouse pointer&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Move the camera - moves the camera or target (depends on movement and rotation mode)&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Set fog visibility - sets visibility distance for basic fog effect&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Set DOF focus - sets object which will appear in focus&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Get Julia constant - gets value of constant for Julia fractal&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Place light #n - places light in selected place (mouse wheel controls distance from fractal) &lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <item>
            <property name="text">
             <string>No action</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButton_multiMode">
              <property name="toolTip">
               <string>Auto - Rotation mode depends on used button</string>
              </property>
              <property name="text">
               <string>Multi</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/navigation/icons/auto.svg</normaloff>:/navigation/icons/auto.svg</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_rotMode">
              <property name="toolTip">
               <string>Rotate camera</string>
              </property>
              <property name="text">
               <string>Rot</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/navigation/icons/rot.svg</normaloff>:/navigation/icons/rot.svg</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_rollMode">
              <property name="toolTip">
               <string>Roll camera</string>
              </property>
              <property name="text">
               <string>Roll</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/navigation/icons/roll.svg</normaloff>:/navigation/icons/roll.svg</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_rotAroundMode">
              <property name="toolTip">
               <string>Rotate around indicated point</string>
              </property>
              <property name="text">
               <string>RotA</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/navigation/icons/rot-around.svg</normaloff>:/navigation/icons/rot-around.svg</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_moveMode">
              <property name="toolTip">
               <string>Move camera</string>
              </property>
              <property name="text">
               <string>Move</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/navigation/icons/cross.svg</normaloff>:/navigation/icons/cross.svg</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_preciseRotation">
           <property name="toolTip">
            <string>Precise rotations</string>
           </property>
           <property name="text">
            <string>P</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/navigation/icons/precise.svg</normaloff>:/navigation/icons/precise.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frameNavigationButtons">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="cDockNavigation" name="widgetNavigationButtons" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonUse">
       <property name="toolTip">
        <string>Closes the Navigator and copies actual parameters to the main window</string>
       </property>
       <property name="text">
        <string>Use in main window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUse_without_camera">
       <property name="toolTip">
        <string>Closes the Navigator and copies actual parameters to the main window. It does not copy camera position and rotation.</string>
       </property>
       <property name="text">
        <string>Use in main window (without camera)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>my_combo_box.h</header>
  </customwidget>
  <customwidget>
   <class>cDockNavigation</class>
   <extends>QWidget</extends>
   <header>dock_navigation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RenderedImage</class>
   <extends>QWidget</extends>
   <header>src/rendered_image_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
